Multisig/App/RemoteNotificationHandler.swift

```diff
@@ -206,18 +173,17 @@ class RemoteNotificationHandler {
     ///   - timestamp: Unix timestamp or nil. If nil, the current time will be used.
     /// - Throws: Error in case of database failures, or private key signing errors
     /// - Returns: If there are any keys, then returns preimage of the hash, the hash that was signed, timestamp used, and array of signatures corresponding to the signing keys.
-    static func sign(safes: [String], deviceID: String, token: String, timestamp: String? = nil) throws -> (preimage: String, hash: String, timestamp: String, signatures: [String])? {
+    static func sign(safes: [String], deviceID: String, token: String, timestamp: String) throws -> (preimage: String, hash: String, signatures: [String])? {
         // sign the registration data by each private key.
 
         let privateKeys = try KeyInfo.keys(types: [.deviceImported, .deviceGenerated])
             .compactMap { try $0.privateKey() }
 
-        guard !privateKeys.isEmpty else {
+        guard !privateKeys.isEmpty,
+              !safes.isEmpty else {
```

Comment:
why do we need this check?

Reply:
because this code is being executed when we import safe without having owner imported